Completion.py

```diff
@@ -96,7 +139,16 @@ def notify_func(filepath, content, callback):
 
 def complete_func(filepath, row, col, content, error_cb, data_cb):
     settings = read_settings()
-    cli = http_client.YcmdClient(settings["server"], settings["port"], settings["hmac"])
```

Comment:
I will maybe attack this later but for now I just left this as is since it works and I dont have the time to change it today or in the next couple days